src/NuGet.Clients/PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -133,9 +133,14 @@ private static bool IsUILegalDisclaimerSuppressed()
             return RegistrySettingUtility.GetBooleanSetting(Constants.SuppressUIDisclaimerRegistryName);
         }
 
-        protected static DependencyBehavior GetDependencyBehaviorFromConfig(
+        protected static DependencyBehavior? GetDependencyBehaviorFromConfig(
             Configuration.ISettings nugetSettings)
         {
+            if (nugetSettings == null)
```

Comment:
It's already returning null at the bottom by default, you can just add this if around the method body to make things more clear